docs/source/subgraphs.md

```diff
@@ -1,82 +1,51 @@
 ---
 title: 'Working with federated graphs'
 sidebar_title: 'Federated graphs'
-description: 'How to use Rover to work with federated graphs and their implementing services'
+description: '(Graphs represented by multiple implementing services)'
```

Comment:
I'm not sure how I feel about the implementing services language. It's technically the most correct, but we've gone back and forth a lot over the terminology of `subgraph` vs implementing services.

Since the command is `subgraph`, do you think it may be best to refer to them as subgraphs? Or since `services` is what's used in Studio, do you think it's still best to call them implementing services?

Reply:
Tweaking this to "Graphs composed of multiple subgraphs" for now.

I think we should leave the service-centric language as-is for now within the article body, since the only representation of this at the moment is implementing services. But we can revisit when the story changes!